integrations/helm/chartsync/chartsync.go

```diff
@@ -48,7 +48,7 @@ import (
 	google_protobuf "github.com/golang/protobuf/ptypes/any"
 	"github.com/google/go-cmp/cmp"
 	"github.com/ncabatoff/go-seq/seq"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
This is an unnecessary alias -- consider reconfiguring your editor (is it VC?)

Reply:
I agree it's useless. Both VIM and VCS are doing that... Thanks to you I had to resolve to using `nano` 😂 

Gone.